kubernetes/deployments_kubeclient.go

```diff
@@ -1311,15 +1359,15 @@ func findOldestAdmittedIngress(ingresses []interface{}) (ingress map[interface{}
 	for idx := range ingresses {
 		ingress, ok := ingresses[idx].(map[interface{}]interface{})
 		if !ok {
-			return nil, errors.New("Bad ingress found in route")
+			return nil, errs.Errorf("bad ingress found in route")
 		}
 		// Check for oldest admitted ingress
 		conditions, ok := ingress["conditions"].([]interface{})
 		if ok {
 			for condIdx := range conditions {
 				condition, ok := conditions[condIdx].(map[interface{}]interface{})
 				if !ok {
-					return nil, errors.New("Bad condition for ingress")
+					return nil, errs.Errorf("bad condition for ingress")
```

Comment:
I think there's also a `errs.New()` function. I mean you're not formatting so no need for `Errorf`.

Reply:
Fixed in several places.